src/Cmd/Add.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include "../Cli.hpp"
+
+#include <string>
+
+extern const Subcmd ADD_CMD;
+std::string getDependencyGitUrl(const std::string& dep);
+std::string getDependencyName(const std::string& dep);
```

Comment:
Do we want to export these functions? If not, we should delete these lines and make these functions within Add.cc static.